pandas/tests/tools/test_to_datetime.py

```diff
@@ -132,16 +132,16 @@ def test_to_datetime_format_YYYYMMDD_with_nat(self, cache):
         # string with NaT
         ser2 = ser.apply(str)
         ser2[2] = "nat"
-        result = to_datetime(ser2, format="%Y%m%d", cache=cache)
-        tm.assert_series_equal(result, expected)
+        with pytest.raises(ValueError, match=None):
+            to_datetime(ser2, format="%Y%m%d", cache=cache)
```

Comment:
this wasn't right to begin with https://github.com/pandas-dev/pandas/issues/50051